gradio/blocks.py

```diff
@@ -1916,7 +1904,7 @@ async def process_api(
             ]
             data, changed_state_ids = zip(*data_and_changed_state_ids)
             if root_path is not None:
-                data = processing_utils.add_root_url(data, root_path, None)
+                data = processing_utils.add_root_url(data, root_path, None)  # type: ignore
```

Comment:
Removing the `Any` return type hint from `postprocess_data` makes pyright complain about this line. I think it does not understand the `zip(*...)` in line 1905.